style.css

```diff
@@ -1,71 +1,71 @@
 @import url('https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
 * {
-    font-family: 'Inter', sans-serif;
-    font-family: 'Poppins', sans-serif;
-    margin: 0;
-    padding: 0;
-    text-decoration: none;
-    box-sizing: border-box;
-    overflow-x: hidden;
-  }
-  body {
-    
-    font-family: 'Poppins', sans-serif;
-    font-family: 'Inter', sans-serif;
-  }
-  
-  header{
-    color:#FFFFFF;
-    width: 100%;
-    position: relative;
-    display: flex;
-    justify-content: end;
-    
-  }
-.menu{
-    padding: 65px 17px;
-    margin-top: 25px;
-    width: auto;
-    
-  }
-  section{
-    background-image: url('images/header-illsutration-mobile.png');
-    background-repeat: no-repeat;
-    background-size: cover;
-    width: 375.30px;
-    height: 653.32px;
-    
-    
+  font-family: 'Inter', sans-serif;
+  font-family: 'Poppins', sans-serif;
+  margin: 0;
+  padding: 0;
+  text-decoration: none;
+  box-sizing: border-box;
+  overflow-x: hidden;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  font-family: 'Inter', sans-serif;
+}
+
+header {
+  color: #fff;
+  width: 100%;
+  position: relative;
+  display: flex;
+  justify-content: end;
+}
+
+.menu {
+  padding: 65px 17px;
+  margin-top: 25px;
+  width: auto;
+}
+
+section {
+  background-image: url('images/header-illsutration-mobile.png');
+  background-repeat: no-repeat;
+  background-size: cover;
+  width: 375.3px;
+  height: 653.32px;
+}
+
```

Comment:
## Styling ✨✨

- [ ] Kindly, it would be better to replicate the given ```spacing``` from the Figma design on your elements for the best user experience 😎😎

   Replicating the `spacing` from the Figma design will help ensure that the final product looks consistent and polished.  😎😎

- [ ] The social media icons are not visible 😨 😨, Please make this small adjustment to replicate the Figma design as much as possible 👍  


![image](https://github.com/AhmedMh0/Mobile-Portfolio/assets/102757126/79965d36-714b-469d-a4a1-19a6646acab2)

Reply:
![Screenshot 2023-06-07 192205](https://github.com/AhmedMh0/Mobile-Portfolio/assets/80107996/fed54214-c9f8-4974-bdc0-1694fbdccacc)
Hey @TuyishimireEric , The icons is shown in my browser! why doesn't it in your browser? can you advise me .